edb/server/server.py

```diff
@@ -954,6 +954,7 @@ async def start(self):
             ri = {
                 "port": self._listen_port,
                 "runstate_dir": str(self._runstate_dir),
+                "tls_cert_file": self._tls_cert_file,
```

Comment:
This is OK, but we should fix the callers to use the non-deprecated option below.

Reply:
Oh cool! Was still wondering why we have both echo and emit. Will fix the CLI test laters :+1: